### PR TITLE
set correct user role in different case

### PR DESCRIPTION
**Pull request by junos**
## Tracker Ticket

* https://quandl.atlassian.net/browse/AP-2607

## Description

* Excel Limits not working properly

## Note to the reviewer

* No

## Note to QA

* None

## Dependency

* None

### CODE REVIEW

File: QuandlExcelAddinWindows/UI/Settings/Settings.xaml.cs [hunk 1]
```diff
@@ -42,6 +42,8 @@ private void SaveSettings()
             QuandlConfig.LongRunningQueryWarning = (bool)LongRunningWarningTextBox.IsChecked;
             QuandlConfig.OverwriteDataWarning = (bool)OverwriteWarningTextBox.IsChecked;
             QuandlConfig.ScrollOnInsert = ScollEnabledCheckBox.IsChecked.Value;
+            if (QuandlConfig.ApiKey.Equals(""))
+                QuandlConfig.UserRole = "";
```
Comment (mbasset) [inline QuandlExcelAddinWindows/UI/Settings/Settings.xaml.cs]: Should we not set user role to null in this case?

File: Quandl.Shared.Modules/QuandlConfig.cs [hunk 2]
```diff
@@ -157,7 +163,9 @@ private static bool RegistryKeyExists(string key)
             try
             {
                 var user = await new Web().WhoAmI(apiKey);
-                return user != null && user.ApiKey == apiKey;
+                var isValid = user != null && user.ApiKey == apiKey;
```
Comment (mbasset) [inline Quandl.Shared.Modules/QuandlConfig.cs]: Might want to use some brackets to make this easier to read.

**Comment by junos** [comment]
@mbasset PR is updated

**Comment by mbasset** [comment]
👍  Looks good but needs QA. Can you create a pre-release branch for QA'ing this?

**Pull request merged**